docs/conformance.md

```diff
@@ -26,11 +26,11 @@ the SPDX community, it has been preserved in this specification.
 
 Profile is the term for a compliance point within the SPDX community across The
 Linux Foundation and OMG. The System Package Data Exchange (SPDX) specification
-defines the following six compliance points, defined as “Profiles”:
+defines the following nine compliance points, defined as “Profiles”:
```

Comment:
"ten" -- why have Core and Software together?

Reply:
@zvr 
> "ten" -- why have Core and Software together?

There are 8 lines, the first line having 2 profiles, so 9 profiles, not 10.

This being said, I agree that we should have only one profile per line.

Reply:
1. Core Profile
1. Software Profile
1. Security Profile
1. Licensing Profile
1. Dataset Profile
1. AI Profile
1. Build Profile
1. Lite Profile
1. Extension Profile